cloud/cmd/admission/admission.go

```diff
@@ -1,3 +1,19 @@
+/*
+Copyright 2020 The KubeEdge Authors.
```

Comment:
The year filed here indicates the date when the file was created, which also indicates the history of the project. So it should be consistent with the created date. Most files was added in 2018, some in 2019, others 2020.

Reply:
I will use `git log` to get first time of file, since linux do not store create time of file.